replicaset.go

```diff
@@ -51,7 +52,7 @@ var (
 
 // doAttemptInitiate will attempt to initiate a mongodb replicaset with each of
 // the given configs, returning as soon as one config is successful.
-func doAttemptInitiate(monotonicSession *mgo.Session, cfg []Config) error {
+func doAttemptInitiate(monotonicSession mgoSession, cfg []Config) error {
```

Comment:
I wish mgo package just had interfaces for things like session and database etc.